src/shogun/evaluation/ClusteringAccuracy.cpp

```diff
@@ -23,6 +23,9 @@ float64_t CClusteringAccuracy::evaluate(CLabels* predicted, CLabels* ground_trut
 	int32_t correct=0;
 	for (int32_t i=0; i < predicted_ilabels.vlen; ++i)
 	{
+		/* print progress */
+		SG_PROGRESS(i, 0, predicted_ilabels.vlen, 1, "EVALUATING ACCURACY: ");
```

Comment:
mmmh I am not too sure about progress bars in evaluation methods, this will take less 1s for every case that your memory can handle ;)

And *actually* this should be vectorized, i.e. shouldnt even be a loop ;)